### PR TITLE
Client, sets up webpack

### PR DESCRIPTION
**Pull request by kmorgan8588**

### CODE REVIEW

**Comment by kmorgan8588** [comment]
This branch was setup before I had circleci merged, just as an fyi

File: client/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+const App = props => {
+    return (
```
Comment (pbennettwa) [inline client/components/App.jsx:5]: I'm guessing that these are placeholders?

**Review by pbennettwa — APPROVED** [review]
This all seems pretty self-explanatory.

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```
Comment (itsOrD) [inline .babelrc]: newline at end

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+public/dist/bundle.js
```
Comment (itsOrD) [inline .gitignore]: newline at end

File: client/components/App.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        <div>
+            <h1>Hello from App!</h1>
+            <p>Tell me more, tell me more</p>
+        </div>
+    );
+}
+
+
+
+export default App;
```
Comment (itsOrD) [inline client/components/App.jsx]: newline at end

File: client/components/App.jsx (hunk 1 above)
Comment (itsOrD) [reply client/components/App.jsx:5]: Seconded?  Surprised this is functional and not stateful, unless you're planning to to use an index.js as your main component and/or utilize hooks?

File: package.json [hunk 5]
```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon server",
-    "db:seed": "node ./seed/index.js"
+    "db:seed": "node ./seed/index.js",
+    "build" : "webpack",
+    "watch" : "webpack -w"
```
Comment (itsOrD) [inline package.json:11]: Interesting choice to differentiate a "build" and a "watch."  What scenario do you foresee using a "build" and not needing an ongoing "watch"?

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <link rel="shortcut icon" href="">
+    <title>Zillow Header</title>
+</head>
+<body>
+    <div id='app'>Hello world from Static!</div>
+    <script src='./dist/bundle.js' type='text/javascript'></script>
+</body>
+</html>
```
Comment (itsOrD) [inline public/index.html]: newline at end

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="shortcut icon" href="">
```
Comment (itsOrD) [inline public/index.html:4]: Not sure what the plan is for this?

File: webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    output: {
+        path: path.resolve(__dirname, 'public', 'dist'),
+        filename: 'bundle.js'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(jsx|js)$/,
+                exclude: /node_modules/, 
+                use: 'babel-loader'
```
Comment (itsOrD) [inline webpack.config.js:15]: I am not entirely clear on the nuances of this difference, but Josh indicated this was a 'better' way to differentiate the rules:use:  

use: [
          {
            loader: 'babel-loader',
            options: {
              presets: ['@babel/preset-env', '@babel/preset-react'],
            },
          },
        ],

**Review by itsOrD — CHANGES_REQUESTED** [review]
Looks like you're ready for some front-end building!

File: client/components/App.jsx (hunk 1 above)
Comment (kmorgan8588) [reply client/components/App.jsx:5]: I wanted to render something to the screen, this is most definitely a placeholder

File: package.json (hunk 5 above)
Comment (kmorgan8588) [reply package.json:11]: When I'm done and want to test out the full interaction, there shouldn't need to have webpack running the whole time

File: public/index.html (hunk 7 above)
Comment (kmorgan8588) [reply public/index.html:4]: I was getting a weird 404 in my console.  I googled and it wasn't any real problem, but the suggested solution had this so it wouldn't throw and error anymore.

File: webpack.config.js (hunk 8 above)
Comment (kmorgan8588) [reply webpack.config.js:15]: I have the same presets in my .babelrc file, I'm pretty sure they do the same thing?

**Comment by kmorgan8588** [comment]
@itsOrD , I made the requested fixes

File: public/index.html (hunk 7 above)
Comment (itsOrD) [reply public/index.html:4]: Interesting. Please let me know if you find out more about what was happening

File: webpack.config.js (hunk 8 above)
Comment (itsOrD) [reply webpack.config.js:15]: You're right. I missed that.

**Review by itsOrD — APPROVED** [review]
Missed the babelrc

**Pull request merged**